<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerProperties</class>
 <widget class="QWidget" name="LayerProperties">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layer Properties</string>
  </property>
  <layout class="QFormLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="title"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="opacityLabel">
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KisSliderSpinBox" name="opacitySlider">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="blendModeLabel">
     <property name="text">
      <string>Blending Mode:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="blendMode"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="visibilityLabel">
     <property name="text">
      <string>Visibility:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="visible">
     <property name="text">
      <string>Visible to you</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="censored">
     <property name="text">
      <string>Censor content</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="sketchMode">
     <property name="text">
      <string>Sketch mode for you</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="sketchOpacityLabel">
     <property name="text">
      <string>Sketch Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="KisSliderSpinBox" name="sketchOpacitySlider">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="sketchTintLabel">
     <property name="text">
      <string>Sketch Tint:</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="collaborationLabel">
     <property name="text">
      <string>Collaboration:</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QCheckBox" name="defaultLayer">
     <property name="text">
      <string>Default for new users</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="createdByLabel">
     <property name="text">
      <string>Created by:</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLabel" name="createdBy">
     <property name="text">
      <string notr="true">#0</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <spacer name="buttonsSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="21" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QWidget" name="sketchTintWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="color_widgets::ColorPreview" name="sketchTintPreview" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="sketchTintSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="widgets::GroupedToolButton" name="sketchTintNoneButton">
        <property name="toolTip">
         <string>No tint</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="widgets::GroupedToolButton" name="sketchTintBlueButton">
        <property name="toolTip">
         <string>Blue tint</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="widgets::GroupedToolButton" name="sketchTintChangeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Chooseâ€¦</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextOnly</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QRadioButton" name="alphaPreserve">
     <property name="text">
      <string>Inherit alpha</string>
     </property>
     <property name="icon">
      <iconset theme="drawpile_alpha_off">
       <normaloff>theme:drawpile_alpha_off.svg</normaloff>theme:drawpile_alpha_off.svg</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">alphaGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QRadioButton" name="clip">
     <property name="text">
      <string>Clip to layer below</string>
     </property>
     <property name="icon">
      <iconset theme="drawpile_selection_intersect">
       <normaloff>theme:drawpile_selection_intersect.svg</normaloff>theme:drawpile_selection_intersect.svg</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">alphaGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QRadioButton" name="alphaBlend">
     <property name="text">
      <string>Blend alpha</string>
     </property>
     <property name="icon">
      <iconset theme="drawpile_alpha_on">
       <normaloff>theme:drawpile_alpha_on.svg</normaloff>theme:drawpile_alpha_on.svg</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">alphaGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="colorLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QSpinBox</extends>
   <header>desktop/widgets/kis_slider_spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorPreview</class>
   <extends>QWidget</extends>
   <header location="global">QtColorWidgets/ColorPreview</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>widgets::GroupedToolButton</class>
   <extends>QToolButton</extends>
   <header>desktop/widgets/groupedtoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>opacitySlider</tabstop>
  <tabstop>blendMode</tabstop>
  <tabstop>visible</tabstop>
  <tabstop>censored</tabstop>
  <tabstop>defaultLayer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="alphaGroup"/>
 </buttongroups>
</ui>
